benchmark/torch/a2c/agent.py

```diff
@@ -0,0 +1,49 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
We should keep the same file structure as that in paddle A2C example. Please rename `agent.py` to `atari_agent.py`.